arches/app/views/base.py

```diff
@@ -130,4 +130,19 @@ def get_context_data(self, **kwargs):
         context["resource_map_layers"] = resource_layers
         context["resource_map_sources"] = resource_sources
 
+        all_map_layers = models.MapLayer.objects.all()
+        all_map_sources = models.MapSource.objects.all()
+
+        map_layers = []
+
+        if self.request.user.is_superuser is True:
+            map_layers = all_map_layers
+        else:
+            for m in all_map_layers:
+                if (m.addtomap is True) or (not self.request.user.has_perm("no_access_to_maplayer", m)):
```

Comment:
@csmith-he can you check that `if (m.addtomap is True)` is only applicable to basemaps? Not sure if this is used by overlays? The intention here was for the basemap set as default must be included, but overlays should not even if specifically set to be included.

Reply:
@aj-he It's behaving this way for overlays as well.  I'm guess this will need further work, then?